tests/ecdsa.js

```diff
@@ -22,6 +22,28 @@ function test (binding) {
       t.same(binding.sign(m, d), expected, `sign(${f.m}, ...) == ${f.signature}`)
     })
 
+    fecdsa.lowR.forEach((f) => {
```

Comment:
These tests,  while useful,  are probaly too specific to the `lowR` use case.
We'd want to test:

* `extraEntropy` == undefined
* `extraEntropy` == 0...1
* `extraEntropy` == random (>1, <n)
* `extraEntropy` == n
* `extraEntropy` == 0xfffffffffffff... (max)